apps/molgenis-components/src/components/forms/RowEditFooter.vue

```diff
@@ -10,9 +10,18 @@
     </div>
     <div class="d-flex" style="gap: 0.5rem">
       <slot></slot>
-      <ButtonAlt @click="$emit('cancel')">Close</ButtonAlt>
-      <ButtonOutline @click="$emit('saveDraft')">Save draft</ButtonOutline>
-      <ButtonAction @click="$emit('save')">Save {{ tableName }}</ButtonAction>
+      <ButtonAlt @click="$emit('cancel')"> Close</ButtonAlt>
+      <ButtonOutline @click="$emit('saveDraft')"> Save draft</ButtonOutline>
+      <Tooltip
+        name="disabled-tooltip"
+        :value="
+          isSaveDisabled ? 'Saving is only possible on the last chapter' : ''
+        "
+      >
+        <ButtonAction @click="$emit('save')" :disabled="isSaveDisabled">
+          Save {{ tableName }}
+        </ButtonAction>
+      </Tooltip>
```

Comment:
most wizards use a next (and prev) button , then at the end some kind of submit button ( or add , buy ). Having a save and disabling the button seems like a strange work around.

Reply:
Agree that there should be a next and prev button instead of the pagination, but I would like to do that in a next PR.
Would you rather have the button not showing? I think that would confuse people as to why there is no save button. 
As for the naming, that should be discussed, but I think we should not focus of on that in this PR.

Reply:
no, would suggest as described above, ux should be part of the story imo. if not possible disabled + tooltip is better then not showing imo ( as tooltip explains the issue). 'Save' is fine as basic action label , but having a wizard guide you through the section would suggest a final closing actions to me , like 'Add' or 'Edit'. Not sure i would suggest blocking ion this but the whole 'disabled' + tooltip feels like trying to optimise a suboptimal solution to me. Maybe the 'why' part of the user story is unclear to me , whats problem is this PR trying to fix ?